README.md

```diff
@@ -56,6 +56,8 @@ You can also [download the latest version of the package][latest-release] and co
 
 ## Usage
 
+> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md) before using icons from Simple Icons.
```

Comment:
I think the link should be to `master` now that the `DISCLAIMER.md` file available there

```suggestion
> :information_source: We ask that all users read our [legal disclaimer](https://github.com/simple-icons/simple-icons/blob/master/DISCLAIMER.md) before using icons from Simple Icons.
```